### DISCUSSION TITLE
Old renovate images are frequently updated on docker hub

### DISCUSSION THREAD

**Discussion by bergerst** [renovatebot/renovate#21439] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

Is there any reason why old renovate images are pushed frequently to dockerhub?

For example, the 35.35.x images were pushed 1-2 days ago while the last commit happened 6 days ago.

![image](https://user-images.githubusercontent.com/24389548/231390092-141667a6-1e0b-45c5-bffc-cab021659147.png)

**Comment by HonkingGoose** [comment]
@viceice I think you know a lot about our `renovate` docker images. Can you see why our CI pushes new images to "old" tags?

**Reply by viceice** [reply]
we rebuild the last 10 versions when the base image changes or some other tools (like node) changed

**Reply by viceice** [reply]
@rarkins maybe it's time to not rebuild old existing versions and just rebuild missing and latest release 🤔

**Reply by viceice** [reply]
- https://github.com/containerbase/internal-tools/issues/332

@rarkins maybe we shouldn't rebuild older missing renovate images at all? 🤔

**Reply by viceice** [reply]
- https://github.com/renovatebot/docker-renovate/pull/1300
- https://github.com/renovatebot/docker-renovate-full/pull/1676
